gcovr/templates/root_page.html

```diff
@@ -120,6 +128,9 @@
     {% if SHOW_DECISION %}
     <th scope="colgroup" colspan=2>Decisions</th>
     {% endif %}
+    {% if SHOW_CALL %}
```

Comment:
Oh, colgroup is missing if option is active. This is also missing for the decision part.
Can you add the optional  colgroups?

Reply:
Added optional colgroups at https://github.com/gcovr/gcovr/commit/fae82e7167833594d497ddb459ab03001fdbab18